.github/workflows/e2e-test.yml

```diff
@@ -49,22 +49,16 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
SPFx doesn't support Node.js v18 yet, will this change break SPFx e2e tests?

Reply:
just let the test cases fail, i will create bugs to fix the e2e case later.